src/librustc_data_structures/indexed_set.rs

```diff
@@ -59,10 +59,6 @@ impl<T: Idx> rustc_serialize::Decodable for IdxSetBuf<T> {
 
 // pnkfelix wants to have this be `IdxSet<T>([Word]) and then pass
 // around `&mut IdxSet<T>` or `&IdxSet<T>`.
-//
-// WARNING: Mapping a `&IdxSetBuf<T>` to `&IdxSet<T>` (at least today)
-// requires a transmute relying on representation guarantees that may
-// not hold in the future.
```

Comment:
Can we make the `IdxSet` `#[transparent]` to actually solve this issue?

Reply:
I'm not familiar with this attribute; is it sufficient that I additionally apply it to `IdxSet`?

Reply:
yes, just apply it to `IdxSet` and we should be good.

Reply:
sorry, it's `#[repr(transparent)]`

Reply:
Ah, that one I do know 👍.